CMakeLists.txt

```diff
@@ -24,40 +23,14 @@ if(UNIX)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -pedantic")
 endif()
 
-#########################
-# External dependencies
-if(PKG_CONFIG_FOUND)
-  search_for_eigen()
-else()
-  find_path(Eigen_INCLUDE_DIR NAMES signature_of_eigen3_matrix_library
-    PATHS
-    ${CMAKE_INSTALL_PREFIX}/include
-    PATH_SUFFIXES eigen3 eigen
-  )
-  include_directories(SYSTEM ${Eigen_INCLUDE_DIR})
-endif()
-
-#########################
+find_package(GUROBI REQUIRED)
+find_package(Eigen REQUIRED)
```

Comment:
Our internal `search_for_eigen()` does additional things related to our toolchain. What problems did you face with it that lead you to this change? I'd rather we fix it in [jrl-umi3218/jrl-cmakemodules](/jrl-umi3218/jrl-cmakemodules)